content/migration-from-otrs-6.rst

```diff
@@ -266,19 +266,32 @@ After creating the temporary copy, all relevant OTRS tables can be moved into th
     Moving tables from one schema to another makes the source schema unusable.
     So make sure the OTRS database used really is a throwaway copy of the productive OTRS database.
 
-First of all, we need a dump of the source OTRS database.
-When ``mysqldump`` is installed and a database connection is possible,
-you can create the database dump directly on the Docker host.
-Alternatively, the database can be dumped on another server and be transferred to the Docker host afterwards.
+First of all, we need a dump of the source OTRS database. As the imported table are copied
+into the OTOBO database we also have to make sure that the character set is converted to *utf8mb4*.
+The dump is split up in *otrs_schema.sql* and *otrs_data.sql* so that the conversion can be
+done in a safe way.
```

Comment:
(table => tables; 
"into the OTOBO database we also" => "into the OTOBO database, we also" 
split up in => split up into)

First of all, we need a dump of the source OTRS database. As the imported tables are copied
into the OTOBO database, we also have to make sure that the character set is converted to *utf8mb4*.
The dump is split up into *otrs_schema.sql* and *otrs_data.sql* so that the conversion can be
done in a safe way.

Reply:
Danke, habe ich in master korrigiert.